plugins/search-backend-module-techdocs/src/collators/DefaultTechDocsCollatorFactory.ts

```diff
@@ -224,17 +226,17 @@ export class DefaultTechDocsCollatorFactory implements DocumentCollatorFactory {
               ]);
 
               return searchIndex.docs.map((doc: MkSearchIndexDoc) => ({
-                ...this.entityTransformer(entity),
-                title: unescape(doc.title),
-                text: unescape(doc.text || ''),
+                ...defaultTechDocsCollatorEntityTransformer(entity),
+                ...defaultTechDocsCollatorDocumentTransformer(doc),
+                ...(this.entityTransformer(entity) ?? {}),
```

Comment:
```suggestion
                ...this.entityTransformer(entity),
```

Reply:
See comment above about Void return type - now the 'default' noop function returns an empty object